src/Xamarin.Android.Build.Tasks/Tasks/LinkApplicationSharedLibraries.cs

```diff
@@ -43,6 +46,10 @@ sealed class InputFiles
 		[Required]
 		public string AndroidBinUtilsDirectory { get; set; }
 
+		public string AndroidNdkDirectory { get; set; }
```

Comment:
Does `EnableMarshalMethodTracing` *require* the NDK, and not just the clang build tools we already redistribute?

Reply:
*oh*, it's to find `libunwind.a`!

Reply:
Not only `libunwind.a`, also a few others from the compiler "private" collection (`libc++abi.a`, `libclang_rt.builtins-<ARCH>-android.a`)

Reply:
Tracing no longer requires NDK to be present on the end user's machine